scripts/gen-s-parser.py

```diff
@@ -591,8 +591,8 @@
     ("struct.set",           "makeStructSet(s)"),
     ("array.new",            "makeArrayNew(s, false)"),
     ("array.new_default",    "makeArrayNew(s, true)"),
-    ("array.new_data",       "makeArrayNewSeg(s, NewData)"),
-    ("array.new_elem",       "makeArrayNewSeg(s, NewElem)"),
+    ("array.new_data",       "makeArrayNewSegData(s)"),
+    ("array.new_elem",       "makeArrayNewSegElem(s)"),
```

Comment:
We can probably remove `Seg` from the class names since it doesn't appear in the instruction names.